dist/index.js

```diff
@@ -4562,7 +4562,8 @@ function run() {
             // We do not want to allow the "default" head state of the package to
             // to be present in the workspace, and colcon will fail stating it found twice
             // a package with an identical name.
-            yield execBashCommand(`find "${rosWorkspaceDir}" -type d -and -name "${repo["repo"]}" | xargs rm -rf`, commandPrefix);
+            const posixRosWorkspaceDir = process.platform !== 'win32' ? rosWorkspaceDir : rosWorkspaceDir.replace(/\\/g, '/');
```

Comment:
I think it might be easier to read if the logic were inverted:
```typescript
const posixRosWorkspaceDir = process.platform === 'win32' ? rosWorkspaceDir.replace(/\\g, '/') : rosWorkspaceDir;
```

Reply:
Done. I switched the order.

Reply:
Thanks,

Off topic, but would you be interested in joining the tooling working group?

Reply:
@zmichaels11 Sure thing.